.vitepress/theme/components/PreferenceSwitch.vue

```diff
@@ -168,7 +168,13 @@ function useToggleFn(
 
 .switch-container {
   display: flex;
-  align-items: center;
+  align-items: center;  
+}
+
+@media(max-width:960px){
+  .switch-container {
+    padding:0px 1em;
```

Comment:
```suggestion
@media(max-width: 959px){
  .switch-container {
    padding: 0 1em;
```

I've tweaked a few things:
1. Spaces after colons.
2. No units for `0`.
3. `max-width: 960px` leads to an inconsistency when the viewport is exactly 960px wide. I've changed the media query to use 959px instead.